sdk/eventhub/event-hubs/tsconfig.json

```diff
@@ -1,40 +1,11 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    /* Basic Options */
-    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
-    "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-
-    "declaration": true /* Generates corresponding '.d.ts' file. */,
-    "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
-    "sourceMap": true /* Generates corresponding '.map' file. */,
-    "inlineSources": true,
-
-    "outDir": "./dist-esm" /* Redirect output structure to the directory. */,
-    "declarationDir": "./types" /* Output directory for generated declaration files.*/,
-
-    "importHelpers": true /* Import emit helpers from 'tslib'. */,
-
-    /* Strict Type-Checking Options */
-    "strict": true /* Enable all strict type-checking options. */,
-    "noImplicitReturns": true /* Report error when not all code paths in function return a value. */,
-
-    /* Additional Checks */
-    "noUnusedLocals": true /* Report errors on unused locals. */,
-
-    /* Module Resolution Options */
-    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-    "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
-
-    /* Experimental Options */
-    "forceConsistentCasingInFileNames": true,
-
-    /* Other options */
-    "newLine": "LF" /*	Use the specified end of line sequence to be used when emitting files: "crlf" (windows) or "lf" (unix).”*/,
-    "allowJs": false /* Don't allow JavaScript files to be compiled.*/,
-    "resolveJsonModule": true
+    "target": "es5",
```

Comment:
does everything need to support IE11 now? 😞

Reply:
_Fixing this, thank you for your patience. 🌼_